venia-integration-tests/src/actions/index.js

```diff
@@ -11,3 +11,4 @@ export * as pageBuilder from './pageBuilder';
 export * as productPage from './productPage';
 export * as toast from './toast';
 export * as wishlistPage from './wishlistPage';
+export * as customerPage from './customerPage';
```

Comment:
Instead of `customerPage`, could we have `footer` since there are other links than Customer Service?
We also are using the footer fields so, I think it would make more sense.

Reply:
Sounds good!